docs/md/Command-line-arguments.md

```diff
@@ -61,8 +61,8 @@ Anything that is not recognized as a known option is interpreted as a file path
 
 ## Options related to forward/inverse search (for LaTeX editors)
 
-- `-forward-search <sourcepath> <line>` : performs a forward search from a LaTeX source file to a loaded PDF document (using PdfSync or SyncTeX). This is an alternative to the ForwardSearch DDE command. E.g. -forward-search main.tex 123 highlights all text related to line 123 in main.tex.
-- `-reuse-instance` : tells an already open SumatraPDF to load the indicated files. If there are several running instances, behavior is undefined. Only needed when communicating with SumatraPDF through DDE (use the ReuseInstance setting instead otherwise).
+- `-forward-search "<sourcepath>" <line> "<pdfpath>"`: performs a forward search from a LaTeX source file to a loaded PDF document (using PdfSync or SyncTeX). This is an alternative to the ForwardSearch DDE command. E.g. -forward-search "/path/to/main.tex" 123 "/path/to/main.pdf" highlights all text related to line 123 in main.tex.
+- `-reuse-instance` : tells an already open SumatraPDF to load the indicated files. If there are several running instances, behavior is undefined. Only needed when communicating with SumatraPDF through DDE (use the ReuseInstance setting instead otherwise). OBS! This is a historic option, newer versions of SumatraPDF shall have it set by default.

```

Comment:
I suggest rather than OBS! add line
  
[Deprecated]: Is only needed for some DDE caller methods. Check for and set the `ReuseInstance = true` value in  Advanced Options instead.